### DISCUSSION TITLE
some kind of timestamp on custom operators

### DISCUSSION THREAD

**Discussion by mickeyvanolst** [cables-gl/cables_docs#172] [Feature Requests] [answered]
![image](https://user-images.githubusercontent.com/1071097/155178056-b6daa978-e136-4bb6-8e59-1d916f5f3965.png)

Is there any way to know when these were last saved?

**Comment by pandrr** [comment, marked as answer]
https://cables.gl/settings -> ops tab

**Reply by mickeyvanolst** [reply]
thanks Tom :-)
